scripts/test_chat_api.py

```
@@ -0,0 +1,211 @@
+PostgreSQL 연동 채팅 API 테스트 스크립트
+
+1. 채팅 스트림 테스트
+2. 세션 조회 테스트
+3. 메시지 히스토리 테스트
+"""
+import asyncio
+import aiohttp
+import json
+import sys
+import os
+from typing import Dict, Any
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)